scheduler/ecs/ecs.go

```diff
@@ -527,16 +527,18 @@ func (m *Scheduler) loadBalancer(ctx context.Context, app *scheduler.App, p *sch
 	return l, nil
 }
 
-func (m *Scheduler) removeLoadBalancer(ctx context.Context, app string, p string) error {
-	l, err := m.findLoadBalancer(ctx, app, p)
+func (m *Scheduler) removeLoadBalancers(ctx context.Context, app string) error {
```

Comment:
So, `Remove` is called when the entire app is destroyed. Previously, this was in `RemoveProcess`, which is called to remove an individual process in the app (say for example, you had a `worker1` process, but then you changed its name to `worker`, we remove the irrelevant ecs service).